PointerStar/Client/Pages/Room.razor

```diff
@@ -259,4 +272,18 @@
             await ViewModel.SubmitDialogAsync();
         }
     }
+
+    
+    private async Task OnClickClipboard(object sender)
```

Comment:
I would move this to the view model so it can be tested.

Reply:
done